gradle/p2-deployment.gradle

```diff
@@ -1,53 +0,0 @@
-/******************************************************************************
- * Copyright (c) 2016 TypeFox and others.
- * 
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v. 2.0 which is available at
- * http://www.eclipse.org/legal/epl-2.0,
- * or the Eclipse Distribution License v. 1.0 which is available at
- * http://www.eclipse.org/org/documents/edl-v10.php.
- * 
- * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
- ******************************************************************************/
-
-apply plugin: 'io.typefox.gradle.p2gen'
```

Comment:
plugin is still loaded in build.gradle

Reply:
I think I removed it: https://github.com/eclipse/lsp4j/pull/378/files#diff-c197962302397baf3a4cc36463dce5eaL27

Reply:
i mean this one https://github.com/eclipse/lsp4j/blob/ec16cc8b2dfa797085be596fc6dd743ea5a4bc65/build.gradle#L20

Reply:
Removed too: https://github.com/eclipse/lsp4j/pull/378/files#diff-c197962302397baf3a4cc36463dce5eaL20

Reply:
PS I don't know how you made your link to the source code look pretty (with inlined source) and mine are ugly links.

Reply:
i used the rightklick copy permalink

Reply:
Thanks - that isn't an option in the diff viewer, but for completeness on this resolved conversation here is the link in the new file showing it gone:

https://github.com/eclipse/lsp4j/blob/fe5212dd8a2a7bed66cc30c669a9aac58226ef34/build.gradle#L18-L20